### PR TITLE
Basic devcontainer

### PR DESCRIPTION
**Pull request by vncoelho**
go to https://github.com/neo-project/neo-devpack-dotnet/tree/devcontainer

Click on

image

You will see

image

### CODE REVIEW

**Comment by Jim8y** [comment]
How do we know if this works?

**Comment by cschuchardt88** [comment]
@vncoelho You not delete the PR `fill out` `form` please. We need it for tracking.

**Comment by vncoelho** [comment]
> @vncoelho Can you not delete the PR `fill out` `form` please? We need it for tracking.

I did not delete. I did not appear in the way I created I think.
I will update later

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+	"name": "C# (.NET)",
+	"image": "mcr.microsoft.com/devcontainers/dotnet:1-7.0-jammy",
+	"onCreateCommand": "./scripts/load_submodule.sh",
```
Comment (shargon) [inline .devcontainer/devcontainer.json:6]: Where is this script?

Comment (vncoelho) [reply .devcontainer/devcontainer.json:6]: I think this script is from nep17 pr, I thought was in master.

**Review by shargon — APPROVED** [review]
We can try

**Comment by vncoelho** [comment]
This is a basic setup, fell free to improve this basic one in another PRs.

**Comment by vncoelho** [comment]
> @vncoelho Can you not delete the PR `fill out` `form` please? We need it for tracking.

I think the form does not appear when we click on a recent submitted branch in order to create the PR. It added another automatic info and skip this.

Comment was added.

**Comment by cschuchardt88** [comment]
> This is a basic setup, fell free to improve this basic one in another PRs.

Sorry, didn't see which repo we were on... Can we have description of the changes and what it is?

**Comment by vncoelho** [comment]
> > This is a basic setup, fell free to improve this basic one in another PRs.
> 
> Sorry, didn't see which repo we were on... Can we have description of the changes and what it is?

Check the comments on the PR

**Pull request merged**